Model/Api/CreateOrder.php

```diff
@@ -638,7 +649,7 @@ protected function getQtyFromTransaction($transactionItem)
      */
     protected function getShippingAmountFromTransaction($transaction)
     {
-        return $transaction->order->cart->shipping_amount->amount;
+        return (int)@$transaction->order->cart->shipping_amount->amount;
```

Comment:
Please do not use `@`.  It better to use `property_exists` or `isset` and return 0 if it does not exist.

Reply:
done